x-pack/agent/pkg/agent/application/fleet_gateway.go

```diff
@@ -27,6 +27,10 @@ type fleetReporter interface {
 	Events() ([]fleetapi.SerializableEvent, func())
 }
 
+type fleetAcker interface {
+	Ack(actionID string) error
+}
+
```

Comment:
Would it be better to use the `Action` interface here instead of the actionID string?